### ISSUE TITLE
Loophole / Escape for (almost) all HTML / CSS based apps? 2.5.8 Pointer Target Spacing

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1354]
Did we create a loophole by mentioning: except IF "Enlarge: A mechanism is available to change the CSS pixel size of each target"

All HTML / CSS based apps can be overwritten and thus are excluded? BUT... This may work for desktop OS / UA but not mobile.

So, do we need to mention this specifically?
Mobile use is one of the biggest issues (the use with a mobile) we try to solve here.

**Comment by jake-abma** [comment]
To expand on this one:

Mentioning that Stylus can change the spacing on a laptop, will this justify the problem experienced on a mobile device where Stylus is not usable?

**Comment by patrickhlauke** [comment]
is stylus (as a third party separate solution) a "mechanism"?

i'd have thought zooming (natively in the browser) would be mechanism though, and that's possible even on mobile (unless the page tries to block this, which would then fail Text Resize anyway)

**Comment by jake-abma** [comment]
@patrickhlauke 

Zooming is not allowed:

> The requirement is independent of the zoom factor of the page, when users zoom in the CSS pixel size of elements does not change. This means that authors cannot meet it by claiming that the target will have enough spacing or sufficient size if the user zooms into the page.

Furthermore I mean just overwriting CSS with your own CSS / file OR use an extension (doing the same?!) (and Stylus might  just be 'a' way)

**Comment by patrickhlauke** [comment]
> Zooming is not allowed

ah, i should actually go look at the new SC text it seems. sounds rather silly though, as in practice users will resort to zooming, and in the real world that *does* increase target size and spacing...

personally not convinced that something that essentially modifies/forces new styles onto content is a "mechanism" - as otherwise lots of other existing SCs that talk about "a mechanism is available" could claim "just use scripting or whatever to hack the current HTML/CSS" (which is essentially what Stylus and co do)

**Comment by jake-abma** [comment]
so what is a proper mechanism according to you when we say:

> Enlarge: A mechanism is available to change the CSS pixel size of each target, or its spacing

?

**Comment by patrickhlauke** [comment]
yes, author supplied buttons, or something in user account/preferences, or a button/switch like MS Office (touch/mouse mode) or similar.

> ![Screenshot of Microsoft Office's 'Touch/Mouse mode' dropdown, and a comparison of (part of) the toolbar as it's presented in each mode](https://user-images.githubusercontent.com/895831/91306252-2f279f00-e7a4-11ea-9733-fbd899c3bbf1.png)


mechanism, to me, suggests something that either comes as default in browser, OS (in all OS's, unless it's a site/app specifically only aimed at one OS?), common AT, or something the author themselves provides. not "if you install this third party extension somebody made..."

**Comment by patrickhlauke** [comment]
otherwise, consider something like 1.4.2 Audio Control

> If any audio on a Web page plays automatically for more than 3 seconds, either a mechanism is available to pause or stop the audio [...]

I could just not bother and claim (or make) an extension that simply strips any `<audio>` element from the DOM on load.
